modules/core/02-client/legacy/v100/store.go

```diff
@@ -154,7 +153,8 @@ func pruneSolomachineConsensusStates(clientStore sdk.KVStore) {
 // addConsensusMetadata adds the iteration key and processed height for all tendermint consensus states
 // These keys were not included in the previous release of the IBC module. Adding the iteration keys allows
 // for pruning iteration.
-func addConsensusMetadata(ctx sdk.Context, clientStore sdk.KVStore, cdc codec.BinaryCodec, clientState *ibctmtypes.ClientState) error {
+//nolint:unparam // Though error is aways nil, we need to run this so pruning works
```

Comment:
Why do we need the return argument here? Can't we remove it?

Reply:
we're using the `nakedret` linter, which checks for functions that don't explicitly state what they return.  Or, do you think it's better to not have any return on this function?

Reply:
An error is never returned here. So it seems fine to me to just remove. cc: @damiannolan

Reply:
OK

Reply:
done